### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Sunnatillo**
Fixes typo

### CODE REVIEW

File: jenkins/scripts/clean_resources.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ set -eu
 # Usage:
 #  clean_resources.sh
 #
-CLIENT_VERSION="7.0.0"
+CLIENT_VERSION="v7.0.0"
```
Comment (tuminoid) [inline jenkins/scripts/clean_resources.sh:14]: Either syntax, 7.0.0 or v7.0.0 works. The version listed in PyPI is 7.0.0 so I'd use that.

File: jenkins/scripts/clean_resources.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ cleanup() {
 }
 
 # Install openstack client
-pip install python-openstackclient="${CLIENT_VERSION}"
+pip install python-openstackclient=="${CLIENT_VERSION}"
```
Comment (tuminoid) [inline jenkins/scripts/clean_resources.sh:48]: This is a legit fix tho.

Comment (Sunnatillo) [reply jenkins/scripts/clean_resources.sh:48]: done

**Pull request merged**